website/static/js/_anonymousLogTexts.js

```diff
@@ -0,0 +1,132 @@
+var anonymousLogActions = {
```

Comment:
It would also be more consistent for this to be a `.json` file. `logActionsList.js` appears to be unused code left over from the refactor; it's actually `logActionsList.json` that's being used.

This would also obviate writing this to a built `.json` file.